packages/execution-environments/src/common/endowments/wasm.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Builds a cross-platform `WebAssembly` only consisting of functions present
+ * both in the browser and Node.js.
+ *
+ * @returns An object with a `WebAssembly` property without `compileStreaming`
+ * and `instantiateStreaming`.
+ */
+const createWasm = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { compileStreaming, instantiateStreaming, ...wasm } = WebAssembly;
+  return { WebAssembly: wasm } as const;
+};
+
+const endowmentModule = {
+  names: ['WebAssembly'] as const,
+  factory: createWasm,
+};
+export default endowmentModule;
```

Comment:
Q: do we prefer default exports for this? If so, when do we prefer those 🤔

Reply:
Here I'm using default exports because all of the files in `endowments/` are designed to be imported and processed in the same way in `endowments/index.ts`. All modules are identical, and their names don't matter, so default exports seem appropriate in this case.

tl;dr: We prefer named exports but we can use defaults if it's appropriate.